src/components/QuizAnswer.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div class="answer-card" 
+       :class="{ correct: isCorrect, incorrect: isIncorrect }"
+       @click="selectAnswer">
+    <img :src="answer.image" :alt="answer.text" class="answer-image" />
+    <h3>{{ answer.text }}</h3>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { defineProps, defineEmits } from 'vue';
+import type { Answer } from '@/config/quizConfig';
+
+const props = defineProps<{
+  answer: Answer;
+  isCorrect: boolean;
+  isIncorrect: boolean;
```

Comment:
Do we need to have both, isCorrect and isIncorrect? An answer that isn't correct is automatically incorrect.

Reply:
Maybe also just increase the type of the "Answer" type and include the "isCorrect" attribute in it